composer.json

```
@@ -43,8 +45,10 @@
         "theofidry/alice-data-fixtures": "^1.5.2"
     },
     "conflict": {
-        "doctrine/annotations": "<1.2.7 || >=3.0",
-        "doctrine/dbal": "<2.11"
+        "doctrine/annotations": "<1.13.1 || >=3.0",
+        "doctrine/dbal": "<2.13.1 || ~3.0.0 || >=4.0",
+        "doctrine/mongodb-odm": "<2.2 || >=3.0",
+        "doctrine/orm": "<2.14 || >=3.0"
```

```suggestion
        "doctrine/orm": "<2.14 || >=4.0"
```

`doctrine/orm` 3.x already exists, but it doesn't work with this bundle yet. Let's keep this.

---

On one hand, I have a project with `"doctrine/orm": "^3.0.2",` and `"liip/test-fixtures-bundle": "^2.7.3",`, and it works. I will not be able to install the next release of this bundle.

On other hand, it is not tested yet with this bundle, so it's better to be safe here.

---

It looks like after #269 it's mostly just widening the version constraints.  I hacked at it locally and on a system without PostgreSQL running (because I don't have Docker here to set it all up), I'm only hitting this error with SQLite:

```sh
Doctrine\DBAL\Exception\NotNullConstraintViolationException: An exception occurred while executing a query: SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: liip_user.name
```

So it seems pretty close and maybe just a mapping or setup error.